ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVUtilsTransactionSpec.scala

```diff
@@ -182,10 +174,7 @@ class KVUtilsTransactionSpec extends AnyWordSpec with Matchers with Inside {
           resultA.successfulLogEntry.getPayloadCase shouldEqual DamlLogEntry.PayloadCase.TRANSACTION_ENTRY
 
           resultB.successfulLogEntry.getPayloadCase shouldEqual DamlLogEntry.PayloadCase.TRANSACTION_REJECTION_ENTRY
-          resultB.successfulLogEntry.getTransactionRejectionEntry.getReasonCase shouldEqual DamlTransactionRejectionEntry.ReasonCase.INCONSISTENT
-          resultB.successfulLogEntry.getTransactionRejectionEntry.getInconsistent.getDetails should startWith(
-            "InconsistentKeys"
-          )
```

Comment:
Are we OK with removing these string matches then?

Reply:
Yes because we can assert on the actual type, v2 rejections are more refined type-wise.